src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.html

```diff
@@ -0,0 +1,122 @@
+          <input *ngIf="editing" type="tel" class="form-control" id="phoneNumber" formControlName="phoneNumber" />
+          <p *ngIf="!editing">{{ user.phoneNumber }}</p>
+        </div>
+
+        <h5>Адреса</h5>
+        <div class="form-group col-sm-3">
+          <label for="city" class="form-label">Місто</label>
+          <input *ngIf="editing" type="text" class="form-control" id="city" formControlName="city" />
+          <p *ngIf="!editing">{{ user.city }}</p>
+        </div>
+
+        <div class="house">
+          <div class="form-group col-sm-3">
+            <label for="street" class="form-label">Вулиця</label>
+            <input *ngIf="editing" type="text" class="form-control" id="street" formControlName="street" />
+            <p *ngIf="!editing">{{ user.street }}</p>
+          </div>
+
+          <div class="form-group col-sm-2">
+            <label for="houseNumber" class="form-label">Будинок</label>
+            <input *ngIf="editing" type="text" class="form-control" id="houseNumber" formControlName="houseNumber" />
+            <p *ngIf="!editing">{{ user.houseNumber }}</p>
+          </div>
+
+          <div class="form-group col-sm-2">
+            <label for="houseCorpus" class="form-label">Корпус</label>
+            <input *ngIf="editing" type="text" class="form-control" id="houseCorpus" formControlName="houseCorpus" />
+            <p *ngIf="!editing">{{ user.houseCorpus }}</p>
+          </div>
+
+          <div class="form-group col-sm-2">
+            <label for="entranceNumber" class="form-label">Підїзд</label>
+            <input *ngIf="editing" type="text" class="form-control" id="entranceNumber" formControlName="entranceNumber" />
+            <p *ngIf="!editing">{{ user.entranceNumber }}</p>
+          </div>
+        </div>
+
+        <div class="form-group col-sm-3">
+          <label for="district" class="form-label">Район</label>
+          <input *ngIf="editing" type="text" class="form-control" id="district" formControlName="district" />
+          <p *ngIf="!editing">{{ user.district }}</p>
+        </div>
+
+        <h5>Зв'язок із соцмережами</h5>
+        <div class="google">
+          <span>
+            <img src="http://pngimg.com/uploads/google/google_PNG19635.png" alt="google" />
+          </span>
+          <p>Зв'язати з акаунтом Google</p>
+        </div>
+      </form>
+    </div>
+
+    <div class="submit-btns" *ngIf="editing">
+      <button class="btn btn-outline-success" type="submit" (click)="onCancel()">Скасувати зміни</button>
+
+      <button class="btn btn-success" type="submit" (click)="onSubmit()">Зберегти зміни</button>
+    </div>
+
+    <div class="footer">
+      <div class="footer-text">
+        <h3>Додаткові дії</h3>
+      </div>
+
+      <div class="footer-btns">
+        <button class="btn btn-outline-success">Змінити пароль</button>
+        <button class="btn btn-outline-success">Видалити профіль</button>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
In that file ubs-client-profile-page.component.html fix  grammar and remove extra space

Reply:
fixed